Source/WebCore/platform/graphics/filters/FEColorMatrix.cpp

```diff
@@ -116,6 +116,10 @@ OptionSet<FilterRenderingMode> FEColorMatrix::supportedFilterRenderingModes() co
 #if USE(CORE_IMAGE)
     if (FEColorMatrixCoreImageApplier::supportsCoreImageRendering(*this))
         modes = modes | FilterRenderingMode::Accelerated;
+#endif
+#if USE(GRAPHICS_CONTEXT_FILTERS)
+    if (m_type == FECOLORMATRIX_TYPE_MATRIX)
+        modes = modes | FilterRenderingMode::GraphicsContext;
```

Comment:
This needs the OS-version based `#ifdef` so that we don't use it on downlevels.

Reply:
`#if HAVE(CGSTYLE_COLORMATRIX_BLUR)` is used instead.